core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/Integration.kt

```diff
@@ -152,6 +156,15 @@ internal class Integration(private val notebook: Notebook, private val options:
 
     override fun Builder.onLoaded() {
         if (version != null) {
+            if (enableExperimentalCsv?.toBoolean() == true) {
+                println("Enabling experimental CSV module: dataframe-csv")
```

Comment:
Do you want to keep it? It can't be disabled on user end

Reply:
I think it's good to have it, a good check for the user that the dataframe integration received the special command correctly and activated the dataframe-csv module. 
It's also good to see it in the output of the notebook, to see something experimental is going on, like a small warning.
When the module is included by default I'll remove the message of course